packages/ui/src/pages/OverviewPage/OverviewPage.tsx

```diff
@@ -8,23 +8,43 @@ import { useQuery } from '../../hooks/useQuery';
 import { useQueues } from '../../hooks/useQueues';
 import { links } from '../../utils/links';
 import s from './OverviewPage.module.css';
+import { QueueSortingDropdown } from '../../components/QueueSortingDropdown/QueueSortingDropdown';
+import { QueueSortKey } from '@bull-board/api/typings/app';
 import OverviewDropDownActions from '../../components/OverviewDropDownActions/OverviewDropDownActions';
 
 export const OverviewPage = () => {
   const { t } = useTranslation();
-  const { actions, queues } = useQueues();
+  const { actions, sortedQueues: queues } = useQueues();
```

Comment:
No need for global sorting method.
It can be simplified & co-located on the dashboard page.
```ts
queuesToView.slice(0).sort(...)
or
queuesToView.toSorted(...)
```